spec/ddtrace/contrib/mysql2/patcher_spec.rb

```diff
@@ -0,0 +1,70 @@
+require 'spec_helper'
+
+require 'ddtrace'
+require 'mysql2'
+
+RSpec.describe 'Mysql2::Client patcher' do
+  let(:tracer) { Datadog::Tracer.new(writer: FauxWriter.new) }
+
+  let(:client) do
+    Mysql2::Client.new(
+      host: host,
+      port: port,
+      database: database,
+      username: 'root',
```

Comment:
Sorry to keep doing this to you but we can't have constants here 😅 . This one should be `TEST_MYSQL_USER`, just like the others.